src/language_names.h

```diff
@@ -23,7 +23,7 @@ struct lang_name_t
 };
 
 //! list of all programming languages Uncrustify supports
-enum lang_flag_e
+enum class lang_flag_e
```

Comment:
Suggest we use a name like `E_Language`.  The `E` at the beginning indicates it is an Enum and the rest of the name indicates the function. Also it would be a good idea to separate the `FLAG_XXX` into a separate `E_Language_Flag` enum. What do you think?